Add `appendMultisigTransaction` to webmode (#771)
scale-it/algo-builder: CHANGELOG.md

from `algosdk` and sends it to the network. 
- Added support for `SignedTransaction` object in `executeTx` method in `Runtime`.
- Added verification for secret key signatures in `Runtime`.
#### @algo-builder/web

> - Add `appendMultisigTransaction` to `WebMode` for append signature to multisig transaction in the algosigner.

Reviewer: ```suggestion
- Add `appendSignMultisigTransaction` function  to `WebMode` for appending  signature to multisig transaction in the algosigner.
```

- Added verification for secret key signatures in `Runtime`.
- Added replace2 and replace3 opcode to `runtime`.

#### @algo-builder/web

> - Add `appendSignMultisigTransaction` function to `WebMode` for appending signature to multisig transaction in the algosigner.

Reviewer: ```suggestion
- Added `appendSignMultisigTransaction` function to `WebMode` for appending signature to multisig transaction in the algosigner.
```

> - Add `appendSignMultisigTransaction` function  to `WebMode` for appending  signature to multisig transaction in the algosigner.

Reviewer: ```suggestion
- Add `appendSignMultisigTransaction` function to `WebMode` for appending signature to multisig transaction in the algosigner.
```
